cats_suite_test.go

```diff
@@ -131,35 +127,34 @@ func TestCATS(t *testing.T) {
 	}, func() {
 		os.Remove(assets.NewAssets().DoraZip)
 	})
-
-	rs := []Reporter{}
-
-	if validationError == nil {
-		if Config.GetArtifactsDirectory() != "" {
-			helpers.EnableCFTrace(Config, "CATS")
-			rs = append(rs, helpers.NewJUnitReporter(Config, "CATS"))
-		}
-	}
-
-	reporterConfig := Config.GetReporterConfig()
-
-	if reporterConfig.HoneyCombDataset != "" && reporterConfig.HoneyCombWriteKey != "" {
-		honeyCombClient := client.New(libhoney.Config{
-			WriteKey: reporterConfig.HoneyCombWriteKey,
-			Dataset:  reporterConfig.HoneyCombDataset,
-		})
-
-		globalTags := map[string]interface{}{
-			"run_id":  os.Getenv("RUN_ID"),
-			"env_api": Config.GetApiEndpoint(),
-		}
-
-		honeyCombReporter := honeycomb.New(honeyCombClient)
-		honeyCombReporter.SetGlobalTags(globalTags)
-		honeyCombReporter.SetCustomTags(reporterConfig.CustomTags)
-
-		rs = append(rs, honeyCombReporter)
-	}
-
-	RunSpecsWithDefaultAndCustomReporters(t, "CATS", rs)
+	//rs := []Reporter{}
+	//
+	//if validationError == nil {
+	//	if Config.GetArtifactsDirectory() != "" {
+	//		helpers.EnableCFTrace(Config, "CATS")
+	//		rs = append(rs, helpers.NewJUnitReporter(Config, "CATS"))
+	//	}
+	//}
+	//
+	//reporterConfig := Config.GetReporterConfig()
+	//
+	//if reporterConfig.HoneyCombDataset != "" && reporterConfig.HoneyCombWriteKey != "" {
+	//	honeyCombClient := client.New(libhoney.Config{
+	//		WriteKey: reporterConfig.HoneyCombWriteKey,
+	//		Dataset:  reporterConfig.HoneyCombDataset,
+	//	})
+	//
+	//	globalTags := map[string]interface{}{
+	//		"run_id":  os.Getenv("RUN_ID"),
+	//		"env_api": Config.GetApiEndpoint(),
+	//	}
+	//
+	//	honeyCombReporter := honeycomb.New(honeyCombClient)
+	//	honeyCombReporter.SetGlobalTags(globalTags)
+	//	honeyCombReporter.SetCustomTags(reporterConfig.CustomTags)
+	//
+	//	rs = append(rs, honeyCombReporter)
+	//}
```

Comment:
The honeyCombReporter I think needs to be rewritten to be properly used.  The junit reporter seems to be a flag on the command line now based on https://onsi.github.io/ginkgo/MIGRATING_TO_V2#improved-reporting-infrastructure.  Curious on folks thoughts about this

Reply:
I'd previously made an issue about this: #528. I'm in favor of removing it, but just in case I've started a [conversation](https://cloudfoundry.slack.com/archives/CHJME4Y80/p1655303517190169) in slack.